src/server/manager.rs

```diff
@@ -22,6 +23,8 @@ fn json_error_handler(error: JsonPayloadError, _: &HttpRequest) -> actix_web::Er
 
 // Start REST API server with the desired address
 pub fn run(server_address: &str) {
+    SignallingServer::start();
```

Comment:
Can you explain a bit better why it should start before the http server ?

Reply:
I just thought that if the signaling server started _after_ the http, the frontend could request the signaling server before it's ready - which it's not a big deal, but can be avoided.

Reply:
So, can you add a comment about it ?